README.md

```diff
@@ -883,6 +885,182 @@ The adapter is part of [PouchDB](http://pouchdb.com/) as documented at:
 - <https://pouchdb.com/api.html#create_database>
 - <http://pouchdb.com/faq.html>.
 
+# Sample: Interact with the SQLite database <a name="sample"></a>
+
+The SQLite storage plugin sample allows you to execute SQL statements to interact with the database. The code snippets in this section demonstrate simple plugin tasks including:
+
+* [Open the database and create a table](#openDb)
+* [Add a row (record) to the database](#addRow)
+* [Read rows from the database](#readRow) that match a column value
+* [Remove a row from the database](#removeRow) that matches a column value
+* [Update rows in the database](#updateRow) that match a column value
+* [Close the database](#closeDb)
+
+##Open the database and create a table <a name="openDb"></a>
+
+Call the `openDatabase()` function to get started, passing in the name and location for the database.
+
+```Javascript
+var db = window.sqlitePlugin.openDatabase({ name: 'my.db', location: 'default' }, function () {
```

Comment:
The window.sqlitePlugin.openDatabase success callback function should expect a parameter with the db object.